app/jobs/create_rss_notifications.rb

```diff
@@ -3,7 +3,7 @@ def perform(options = {})
     # Defaults to theforeman.org blog RSS
     UINotifications::RssNotificationsChecker.new(options).deliver!
   ensure
-    self.class.set(:wait => 12.hours).perform_later(options)
+    self.class.set(:wait => 12.hours).perform_later(options.delete_if {|k, v| k =~ /^_aj_/ })
```

Comment:
It's probably safer to use `reject` instead of `delete_if`, especially if this is some internal data activejob might be using later and might be missing, if we delete it from the hash.

Reply:
Also, you don't need to expose `|v|` in the block AFAIK, so `options.reject { |k| k.to_s =~ /^_aj_/ }` should work better (also, it would be good to have `k.to_s`, just in case

Reply:
Tested with `options.reject { |k| k.to_s =~ /^_aj_/ }` and worked smoothly